librz/core/cmd_descs/cmd_zign.yaml

```diff
@@ -185,15 +185,21 @@ commands:
   - name: zf
     summary: Manage FLIRT signatures
     subcommands:
+      - name: zfc
+        cname: zign_flirt_create
+        summary: Create a FLIRT file with .pac or .sig
+        args:
+          - name: filename
+            type: RZ_CMD_ARG_TYPE_FILE
       - name: zfd
         cname: zign_flirt_dump
-        summary: Open FLIRT file and dump
+        summary: Opens a FLIRT file (.pac or .sigà) and dumps its contents
```

Comment:
Before you used "create" now it "opens", please use either. I think it's better to keep "open"